UnityProject/Assets/Scripts/Managers/SubSceneManager/SubSceneManager.Client.cs

```diff
@@ -49,4 +51,36 @@ IEnumerator LoadClientSubScene(SceneInfo sceneInfo)
 
 		clientIsLoadingSubscene = false;
 	}
+
+	public void LoadScenesFromServer(List<SceneInfo> Scenes, string OriginalScene, Action OnFinish)
+	{
+		StartCoroutine(LoadClientScenesFromServer(Scenes,OriginalScene, OnFinish));
+	}
+
+	IEnumerator LoadClientScenesFromServer(List<SceneInfo> Scenes, string OriginalScene, Action OnFinish)
+	{
+		clientIsLoadingSubscene = true;
+		foreach (var Scene in Scenes)
+		{
+			clientIsLoadingSubscene = true;
+			yield return LoadClientSubScene(Scene, false);
+			clientIsLoadingSubscene = true;
+			clientLoadedSubScenes.Add(Scene);
+		}
+
+		NetworkClient.PrepareToSpawnSceneObjects();
+		yield return WaitFor.Seconds(0.2f);
```

Comment:
Oh no, can't we wait until it is reported that all scenes loaded instead of fixed time?

Reply:
I can try removing it I just added it in case it might explode

Reply:
Yeah I'd find the way to remove all of them waiting times